tests/v3/test_codecs.py

```diff
@@ -966,10 +966,10 @@ async def test_blosc_evolve(store: Store):
         codecs=[BytesCodec(), BloscCodec()],
     )
 
-    zarr_json = json.loads(await (store / "blosc_evolve_u1" / "zarr.json").get())
+    zarr_json = await (store / "blosc_evolve_u1" / "zarr.json").get_metadata()
     blosc_configuration_json = zarr_json["codecs"][1]["configuration"]
     assert blosc_configuration_json["typesize"] == 1
-    assert blosc_configuration_json["shuffle"] == "bitshuffle"
+    assert blosc_configuration_json["shuffle"] == BloscShuffle.bitshuffle
```

Comment:
I threw this in because it makes the test pass but its really not what we want. We were previously letting `json.dumps` do a lot of work to serialize Python objects. I think this means that we now need to push more of that logic into the `to_dict()` methods on the Metadata objects.

cc @d-v-b